Source/WebKit/Platform/IPC/Decoder.h

```diff
@@ -185,6 +202,8 @@ class Decoder {
     ImportanceAssertion m_importanceAssertion;
 #endif
 
+    Vector<ClassStructPtr> m_allowedClasses;
```

Comment:
This could be inside PLATFORM(COCOA) since no other platforms decode ObjC types.